### PR TITLE
Add boxcryptor beta 2.3.201.730

### PR DESCRIPTION
**Pull request by blogabe**
#### Adding a new cask

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/boxcryptor-beta.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'boxcryptor-beta' do
+  version '2.3.201.730'
```
Comment (vitorgalvao) [inline Casks/boxcryptor-beta.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by vitorgalvao** [comment]
Please follow the instructions and fill the template.

**Comment by blogabe** [comment]
completed template and fixed style

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**